Classes/Errors/TMAPIError.h

```diff
@@ -46,6 +46,11 @@
  */
 @property (nonatomic, readonly) BOOL needsAge;
 
+/**
+ A specific property associated with some errors for the GUCE Consent flow. This specifies the age the user must be based on their gdpr region.
+ */
+@property (nonatomic, readonly) NSInteger minimumAge;
```

Comment:
Can this property be more accurately named? `minimumAge` doesn't really mean much in this context